Species/Languages.md

```diff
@@ -1,27 +1,117 @@
+
+Common could reasonably be spoken as a trade pidgin anywhere, and human populations where common is rare, or where a different trade pidgin is spoken, should have an explanation.
+%%
+## Drankorian Language Family
+These are languages that derive from Drankorian, and are the native tongues of most of the lands that were once part of the Drankorian Empire. All Drankorian languages are written using the Drankorian alphabet.
+
+After the fall of the Drankorian Empire, the Drankorian language developed into three main languages: Sembaran, proto-Cymean, and proto-Chardonian. The modern languages of Isinguese and Cymean are still somewhat mutually comprehensible, as are the modern languages of Chardonian and Illorian.
+
+_Real world analog: Drankorian and its language family is broadly based on Hellenic and Romance languages, with some small variations_
+
+* **Drankorian**: The ancient language of Drankor, it is not commonly spoken anymore, although it is still a common scholarly tongue. Drankorian words are usually based on classic Latin or Greek.
+* **Sembaran**: A number of mutually understandable related languages and dialects spoken in Sembara and Addermarch, but with a number of loan words from Tollish and Zimkovan. Southern Sembaran is more purely Drankorian, and tends to use French-derived words, whereas northern Sembaran tends to use English. Mutually understandable with Tollish and in some cases, Isinguese. 
+* **Chardonian**: A number of mutually understandable related languages and dialects spoken in the Chardonian Empire, but with some loan words from Northros languages. Self-consciously seen as the inheritors of the Drankorian language, and sometimes has similar sounds. Tends to use Latin or Italian, with the occasional "classical" Greek. 
+* **Cymean**: A number of mutually understandable related languages and dialects spoken in the Cymean peninsula. Cymean words are usually based on Spanish or Portugese, but with some Italian. It is usually understood by speakers of Isinguese. 
+* **Isinguese:** The language spoken in the now destroyed lands of the Istabor Alliance, but still spoken in some of the refugee communities including the [[Duchy of Maseau]] and the [[Refounded Alliance of Aurbez]]. Based on Spanish or French, and is usually understood by speakers of Sembaran (especially southern Sembarans) or Cymean. 
+* **Illorian**: The language spoken on the islands of Illoria, it is often understandable to Chardonian speakers, especially those from the [[Apporia|Apporian Peninsula]] or [[Chardon]] itself. No defined real world mapping yet.
+* **~Northern Dialects~**: A set of now-almost extinct languages spoken before the Great War in a number of the northern realms, such [[Volganis]], [[Amani]], the [[Ausberg Empire]], and the [[Republic of Varjatta]]. These languages were usually understood by speakers of Chardonian, and are based on Italian.
+
+%% Background Note:
```

Comment:
Will give this a little thought if there is anything missing. But I think pending the history of the west cleanup this is probably right.

Reply:
I've given a bit more thought to western history and I think there is a cultural group missing, the people who never came under Drankorian rule but also weren't Deno'qai. Need to think a bit more about this and the notes of a language doc probably aren't the place so resolving this, but more to come eventually.